Ft reset ps 31

tysepa opened the pull request:
This is a reset page 

Here you can reset your password even if you have forgotten 
and when you enter your email you have to create your new password 

How should this be manually tested?
Clone the repo

checkout to branch ft-resetPs-31

Run npm install to install modules

Run npm run start to start the development server

On the landing page click on the forget password on the sign-in page menu

Commit:
ft(Reset):Reset password

On this page the user can get the place to insert their email to rest password
The driver and operator can reset their password
Anyone has access here can provide their email

WIP #31

Commit:
ft(Reset):Reset password

On this page the user can get the place to insert their email to rest password
The driver and operator can reset their password
Anyone has access here can provide their email

WIP #31

Commit:
ft(Reset):Reset password

On this page the user can get the place to insert their email to reset password
The driver and operator can reset their password
Anyone has access here can provide their email

WIP #31

Commit:
ft(chore) : Rest password

update on reset password

WIP #31

Commit:
ft(chore): Reset password 31

The use can provide their email to rest the password

WIP #31

Commit:
updated change

Commit:
ft(Reset) : Reset passowrd

Here you can use your email to reset password
Driver, operator and user can use this page

WIP #31

Commit:
ft(Reset) : Reset passowrd

Here you can use your email to reset password
Driver, operator and user can use this page

WIP #31

Commit:
ft(Reset) : Reset passowrd

Here you can use your email to reset password
Driver, operator and user can use this page

WIP #31

PrinceNiyonshuti commented on tailwind.config.js:
```diff
@@ -7,7 +7,13 @@ module.exports = {
+
+      'sm':'425px',
+      'md':'768px',
+      'xl':'1024px',
```
why you used sm,md,xl,2xl  , cant we use lg too instead of having 2xl

tysepa replied:
but this is work well

PrinceNiyonshuti reviewed (changes requested):
please provide a detailed PR description and review the changes requested

tysepa reviewed (commented):
all done

thepatrickniyo reviewed (commented):
Rename your Pull request for it to be self-explanatory. 

eg: **feat reset password #31**

Commit:
ft(Reset): Reset password

Here you can use your email to reset password
Driver, Operator and user can use this page
Wip #31

Commit:
ft(Reset): Reset password

Here you can use your email to reset password
Driver, Operator and user can use this page
Wip #31

Commit:
ft(Reset): Reset password

Here you can use your email to reset password
Driver, Operator and user can use this page
Wip #31

tysepa reviewed (commented):
I have made the change on this PR

PrinceNiyonshuti reviewed (approved):
looks good

Commit:
ft(Reset): Reset password

Here you can use your email to reset password
Driver, Operator and user can use this page

WIP #31

Commit:
ft(reset): Password

make change on Path

WIP #31

Commit:
ft(Reset): Reset password

Here you can use your email to reset password
Driver, Operator and user can use this page

WIP #31

Commit:
ft(Reset): Reset password

Here you can use your email to reset password
Driver, Operator and user can use this page

WIP #31

Commit:
ft(chore):Reset Password

Make test to page and rise coverage
Can provide your email to reset your password

WIP #31

rachelakimana reviewed (approved):
Looks good

Commit:
ch (setup):Phantom app

Phantom App that helps to navigate the bus
Here is the Landing page for the user
There is the Login page
There is the Register page
All user can provide their email or create a new account
There is a Reset password page that can use to reset your password when you forgot it

Finished WIP #31

Commit:
ch (setup):Phantom app

Phantom App that helps to navigate the bus
Here is the Landing page for the user
There is the Login page
There is the Register page
All user can provide their email or create a new account
There is a Reset password page that can use to reset your password when you forgot it

Finished WIP #31

Commit:
make exclude unneed file in Coverage
